src/EnergyPlus/FluidProperties.cc

```diff
@@ -4593,7 +4575,7 @@ namespace FluidProperties {
 
     int GetGlycolRawNum(EnergyPlusData &state, std::string_view const glycolRawName) // carries in substance name
     {
-        auto &df = state.dataFluidProps;
+        auto const &df = state.dataFluidProps;
```

Comment:
[src/EnergyPlus/FluidProperties.cc:4596]:(style),[constVariable],Variable 'df' can be declared as reference to const